src/main/java/com/amazonaws/cloudhsm/examples/AESWrappingRunner.java

```diff
@@ -71,6 +72,58 @@ public static void main(String[] args) throws Exception {
         aesWrappingKey.destroy();
         payloadKey.destroy();
     }
+    
+    /**
+     * This method demonstrates "AES/GCM/NoPadding"
+     * @param aesWrappingKey AES key for wrapping
+     * @param payloadKey Some payload-key to be wrapped
+     * @throws Exception
+     */
+    private static void aesGcmWrapNoPadding(Key aesWrappingKey, Key payloadKey) throws Exception {
+        // Create an AESWrap no padding cipher from the CloudHSM provider.
+        final Cipher cipher =
+                Cipher.getInstance("AES/GCM/NoPadding", CloudHsmProvider.PROVIDER_NAME);
+
+        //Initialization Vector - this is overridden by HSM
+        SecureRandom random = new SecureRandom();
+        byte[] IV = new byte[12];
+        random.nextBytes(IV);
```

Comment:
Since the IV is set by the HSM. Just set the byte[] to all zeros. No reason to use random here.

```suggestion
        byte[] IV = new byte[12];
```